packages/react-native-editor/package.json

```diff
@@ -1,150 +1,49 @@
-    "test:e2e:build-app:android": "yarn test:e2e:bundle:android && cd android && ./gradlew clean && ./gradlew assembleDebug",
-    "test:e2e:install-app:android": "cd android && ./gradlew installDebug",
-    "test:e2e:bundle:ios": "react-native bundle --reset-cache --platform=ios --dev=false --minify false --entry-file=index.js --bundle-output=./ios/build/gutenberg/Build/Products/Release-iphonesimulator/gutenberg.app/main.jsbundle --assets-dest=./ios/build/gutenberg/Build/Products/Release-iphonesimulator/gutenberg.app",
-    "test:e2e:build-app:ios": "react-native run-ios --configuration Release --no-packager --simulator=\"iPhone 11\" && yarn test:e2e:bundle:ios && WORK_DIR=$(pwd) && cd ./ios/build/gutenberg/Build/Products/Release-iphonesimulator && zip -r $WORK_DIR/ios/Gutenberg.app.zip gutenberg.app",
-    "test:e2e:install-app:ios": "yarn test:e2e:build-app:ios",
-    "flow": "flow",
-    "prettier": "prettier-eslint --write $npm_package_config_jsfiles $npm_package_config_scssfiles",
-    "prettier:check": "prettier-eslint --list-different $npm_package_config_jsfiles $npm_package_config_scssfiles || { echo '\nERROR: `yarn prettier:check` found a formatting problem.\nNo files have been changed. Try running `yarn prettier` to fix any formatting issues.\n'; exit 1; }",
-    "build:gutenberg": "cd gutenberg && npm install && npm run build",
-    "clean": "yarn clean:aztec; yarn cache clean; yarn clean:haste; yarn clean:jest; yarn clean:metro; yarn clean:react; yarn clean:watchman; yarn clean:node; yarn clean:pot",
-    "clean:runtime": "yarn clean:haste; yarn clean:react; yarn clean:metro; yarn clean:jest; yarn clean:watchman; yarn clean:babel-cache",
-    "clean:aztec": "cd react-native-aztec && yarn clean && cd example && yarn clean",
-    "clean:gutenberg": "cd gutenberg && npm run clean:packages && rm -rf build && rm -rf node_modules",
-    "clean:pot": "rm -f gutenberg*.pot",
-    "clean:haste": "rm -rf /tmp/haste-map-react-native-packager-*",
-    "clean:install": "yarn clean; yarn",
-    "clean:jest": "yarn jest --clearCache --config jest.config.js; rm -rf $TMPDIR/jest_*",
-    "clean:jest-inside-gb": "yarn jest --clearCache --config jest_gb.config.js; rm -rf $TMPDIR/jest_*",
-    "clean:metro": "rm -rf $TMPDIR/metro-cache-*;",
-    "clean:react": "rm -rf $TMPDIR/react-*",
-    "clean:node": "rm -rf node_modules",
-    "clean:watchman": "command -v watchman >/dev/null 2>&1 && watchman watch-del-all; true",
-    "clean:babel-cache": "rm -rf ./node_modules/.cache/babel-loader/*",
-    "clean:i18n-cache": "rm -rf ./i18n-cache/data/*.json && rm -f ./i18n-cache/index.native.js",
-    "lint": "eslint . --ext .js",
-    "lint:fix": "yarn lint --fix",
-    "version": "yarn bundle && git add -A bundle"
-  },
   "dependencies": {
     "@babel/runtime": "^7.3.1",
```

Comment:
`@babel/runtime` might need to be aligned with other packages closer to merge. It's `7.8.x` at the moment in `master`.

Reply:
Yep, we'll need to update the feature branch with the latest changes from master. I'll do that in a separate PR (or update directly if there is no conflicts to resolve)

Reply:
Cool 👍